### ISSUE TITLE
Sweet Mars

### ISSUE THREAD

**Issue by codeSnipps** [EliverLara/Sweet#83]
The top bar is popping a white menu and its suposed to be dark

**Comment by EliverLara** [comment]
Hi, could you provide a screenshot please?

**Comment by laneflee** [comment]
![Bildschirmfoto von 2020-05-16 16-53-57](https://user-images.githubusercontent.com/44040196/82122838-781f8e80-9785-11ea-976b-1bb427977e1b.png)

The headers are also white for me. I haven't found a guide to enable dark mode, so I decided to move to gtk-3.0 (no idea how to work with gtk-2.0), turn on my brain, and find the envariable "$variant". Assuming that it is set somewhere deep, I didn't want to mess with it. So, I just moved gtk-dark.css and gtk-dark.scss to gtk.css and gtk.scss respectively. Everything's dark now, except for the top rows (excluding native apps):
![Bildschirmfoto von 2020-05-16 17-04-38](https://user-images.githubusercontent.com/44040196/82123017-976aeb80-9786-11ea-8ea8-a06e0c477324.png)

P.S: tab titles in Firefox are a bit too blended in: it is hard to read them :(

Nonetheless, this theme is one of my favorites! Thank you <3
